README.md

```diff
@@ -138,6 +128,12 @@ An alternative to adding an `odrive_config_file` is running the command `odrivet
 }
 ```
 
+## Next Steps
+
+- To test your ODrive motor, go to the [**Control** tab](https://docs.viam.com/manage/fleet/robots/#control).
+- To write code against your ODrive motor, use one of the [available SDKs](https://docs.viam.com/program/).
+- To view examples using a motor component, explore [these tutorials](https://docs.viam.com/tutorials/).
+  
 ## Troubleshooting
 
```

Comment:
also a question here, is the troubleshooting section now empty? maybe it's just on the website and not in the readme? I remember that having important stuff. if it's only on the online documentation I think we should transfer it here